### PR TITLE
Some minor changes to the app

### PR DESCRIPTION
**Pull request by 9a24f0**
This PR includes:
- Add splash screen to make it look smoother when loading logged session
- Add color indicates which activity is on screen, also disable open RepoActivity from RepoActivity (same with ProfileActivity)
- Align icons on nav drawer, as @PhuongPhg requested

### CODE REVIEW

File: app/src/main/java/vn/edu/usth/minigh/BaseActivity.java [hunk 1]
```diff
@@ -58,11 +56,15 @@ public void redirectActivity(Class aclass) {
     }
 
     public void clickProfile(View view) {
-        redirectActivity(ProfileActivity.class);
+        if (this.getClass() != ProfileActivity.class) {
+            redirectActivity(ProfileActivity.class);
+        }
```
Comment (McSinyx) [inline app/src/main/java/vn/edu/usth/minigh/BaseActivity.java:61]: Nice!

File: app/src/main/java/vn/edu/usth/minigh/SplashActivity.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+import java.util.Timer;
+import java.util.TimerTask;
+
+public class SplashActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_splash);
+    }
```
Comment (McSinyx) [inline app/src/main/java/vn/edu/usth/minigh/SplashActivity.java:17]: I'd prefer overriding the constructor for the ease of inheritance but in this case it does not matter.

**Review by McSinyx — APPROVED** [review]
LGETM!

File: app/src/main/java/vn/edu/usth/minigh/SplashActivity.java (hunk 2 above)
Comment (9a24f0) [reply app/src/main/java/vn/edu/usth/minigh/SplashActivity.java:17]: LOL we have that when we create the activity in Android Studio, let's skip this.

**Pull request merged**